wallet/res/xml/preference_settings.xml

```diff
@@ -1,72 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android" >
```

Comment:
This doesn't seem to be used.

Reply:
Correct.  The various settings menus used to be in these preference files.